iglu-registry/0.3.0/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM snowplow-docker-registry.bintray.io/snowplow/base:0.1.0
+LABEL maintainer="Snowplow Analytics Ltd. <support@snowplowanalytics.com>"
+
+# The version of the registry to download
+ENV IGLU_REGISTRY_VERSION="0.3.0"
+
+# The name of the archive to download
+ENV ARCHIVE="iglu_server_${IGLU_REGISTRY_VERSION}.zip"
+
+# Install Iglu Registry
+RUN mkdir -p /tmp/build && \
+    cd /tmp/build && \
+    wget -q http://dl.bintray.com/snowplow/snowplow-generic/${ARCHIVE} && \
+    unzip -d ${SNOWPLOW_BIN_PATH} ${ARCHIVE} && \
+    cd /tmp && \
+    rm -rf /tmp/build && \
+    mv ${SNOWPLOW_BIN_PATH}/iglu-server-* ${SNOWPLOW_BIN_PATH}/iglu_server.jar && \
+    chown -R snowplow:snowplow ${SNOWPLOW_BIN_PATH}
+
+# Install postgresql
+RUN apk update && \
+    apk add postgresql-client
+
+COPY ./scripts/super_api_key_init.sh ${SNOWPLOW_BIN_PATH}
+RUN chown -R snowplow:snowplow ${SNOWPLOW_BIN_PATH}
```

Comment:
this is already done above

Reply:
Do you mean base image referring to "above" @BenFradet ? This is here because I noticed that, after `COPY` command, user and group permissions of `super_api_key_init.sh` is `root:root`, not `snowplow:snowplow`.

Reply:
the chown is done twice

Reply:
in your dockerfile

Reply:
Oh sorry, I noticed now. Will remove first one.